lib/restivus.coffee

```diff
@@ -307,25 +307,30 @@ class @Restivus
 
       After the user is logged out, the onLoggedOut hook is called (see Restfully.configure() for adding hook).
     ###
+    logout = ->
+      # Remove the given auth token from the user's account
+      authToken = @request.headers['x-auth-token']
+      hashedToken = Accounts._hashLoginToken authToken
+      tokenLocation = self.config.auth.token
+      index = tokenLocation.lastIndexOf '.'
+      tokenPath = tokenLocation.substring 0, index
+      tokenFieldName = tokenLocation.substring index + 1
+      tokenToRemove = {}
+      tokenToRemove[tokenFieldName] = hashedToken
+      tokenRemovalQuery = {}
+      tokenRemovalQuery[tokenPath] = tokenToRemove
+      Meteor.users.update @user._id, {$pull: tokenRemovalQuery}
+
+      # TODO: Add any return data to response as data.extra
+      # Call the logout hook with the logged out user attached
+      self.config.onLoggedOut?.call this
+
+      {status: "success", data: message: 'You\'ve been logged out!'}
+
     @addRoute 'logout', {authRequired: true},
       get: ->
-        # Remove the given auth token from the user's account
-        authToken = @request.headers['x-auth-token']
-        hashedToken = Accounts._hashLoginToken authToken
-        tokenLocation = self.config.auth.token
-        index = tokenLocation.lastIndexOf '.'
-        tokenPath = tokenLocation.substring 0, index
-        tokenFieldName = tokenLocation.substring index + 1
-        tokenToRemove = {}
-        tokenToRemove[tokenFieldName] = hashedToken
-        tokenRemovalQuery = {}
-        tokenRemovalQuery[tokenPath] = tokenToRemove
-        Meteor.users.update @user._id, {$pull: tokenRemovalQuery}
-
-        # TODO: Add any return data to response as data.extra
-        # Call the logout hook with the logged out user attached
-        self.config.onLoggedOut?.call this
-
-        {status: "success", data: message: 'You\'ve been logged out!'}
+        console.error "WARNING: Logout via GET is deprecated. See https://github.com/kahmali/meteor-restivus/issues/100"
```

Comment:
A few small things:

1. This should use `console.warn()` 
1. I think we should inform them here that they should use POST instead
1. It's probably a good idea to print the last sentence (with the link info) on a new line, so users don't have to scroll to see the full output

Here's my suggested output:
```
WARNING: Default logout via GET will be removed in v1.0. Use POST instead.
See https://github.com/kahmali/meteor-restivus/issues/100
```